scripts/libraries/diamond.ts

```diff
@@ -13,7 +13,12 @@ export const FacetCutAction = { Add: 0, Replace: 1, Remove: 2 };
 // get function selectors from ABI
 export function getSelectors(contract: Contract | ContractFactory<any[], BaseContract>): SelectorsObj {
   const signatures = Object.values(
-    contract.interface.fragments.filter((fragment) => fragment.type === "function"),
+    contract.interface.fragments
+      .filter((fragment) => fragment.type === "function")
+      .filter((fragment) => {
+        const functionFragment = fragment as FunctionFragment;
+        return functionFragment.selector && !functionFragment.selector.startsWith("0x00000000");
+      }),
   ) as FunctionFragment[];
   const selectors = signatures.reduce<string[]>((acc, val) => {
     if (val.format("sighash") !== "init(bytes)") {
```

Comment:
```suggestion
    if (!val.format("sighash").startsWith("init")) {
```
We should always filter "init" out of the selectors (regardless of their argument types), since they should not be added as a diamond method.